org.upnproutercontrol.UPnPRouterControl.json

```diff
@@ -0,0 +1,75 @@
+{
+   "app-id":"org.upnproutercontrol.UPnPRouterControl",
+   "branch":"unstable",
+   "runtime":"org.gnome.Platform",
+   "runtime-version":"3.38",
+   "sdk":"org.gnome.Sdk",
+   "command":"upnp-router-control",
+   "rename-desktop-file":"upnp-router-control.desktop",
+   "finish-args":[
+      "--share=network",
+      "--socket=session-bus",
+      "--socket=x11",
```

Comment:
you only need this or fallback-x11 not both, so remove the x11 socket

Reply:
Removed x11 socket